runebender-lib/src/data.rs

```diff
@@ -911,6 +923,25 @@ mod lenses {
             f(&mut s)
         }
     }
+
+    fn find_kern_groups(data: &Workspace, glyph: &Glyph) -> (String, String) {
+        let mut kern1_group = "".to_string();
+        let mut kern2_group = "".to_string();
```

Comment:
I guess a few different things here:

- should these be `Option` instead of `String`?
- you might prefer `String::new()` for creating a new, empty `String`.

At a higher level: I wonder if runebender shouldn't keep track of kern groups via some other mechanism, and only extract / write back to the UFO format as needed? trying to manipulate this directly just feels so awkward.